chain/yoda/execute.go

```diff
@@ -121,6 +121,8 @@ func SubmitReport(c *Context, l *Logger, keyIndex int64, reports []ReportMsgWith
 			switch txRes.Code {
 			case 0:
 				l.Info(":smiling_face_with_sunglasses: Successfully broadcast tx with hash: %s", txHash)
+				c.pendingGauge = c.pendingGauge - int64(len(reports))
+				c.submittedCount++
```

Comment:
I think we want to track submitted reports, so we must add the length of reports?